tests/phpunit/tests/query/searchColumns.php

```diff
@@ -0,0 +1,381 @@
+<?php
+/**
+ * Testing the search columns support in `WP_Query`.
+ *
+ * @package WordPress\UnitTests
+ * @since 6.2.0
+ */
+
+/**
+ * Test cases for the search columns feature.
+ *
+ * @group query
+ * @group search
+ *
```

Comment:
```suggestion
 * @group search
 *
 * @covers WP_Query::parse_search
 *
```
Adding the `@covers` annotation will help improve the accuracy of coverage reports.

Reply:
Added.